api/server/structs/conversions.go

```diff
@@ -1210,7 +1222,7 @@ func AttesterSlashingsElectraToConsensus(src []*AttesterSlashingElectra) ([]*eth
 		if err != nil {
 			return nil, server.NewDecodeError(err, fmt.Sprintf("[%d].Attestation1.Signature", i))
 		}
-		err = slice.VerifyMaxLength(s.Attestation1.AttestingIndices, 2048)
```

Comment:
was this value wrong before?

Reply:
Yes, it was. It's 131072 now. See this section:
https://notes.ethereum.org/@jtraglia/prysm_nits#Incorrect-attester-slashing-length-checks